src/utils/queue.js

```diff
@@ -23,9 +24,18 @@ class Queue extends EventEmitter {
     }
 
     /**
-     * 
-     * 
-     * @param {any} item 
+     * @param {Object} item 
+     * @param {Number} item.item
```

Comment:
While currently the queue is only used for queuing cluster connects / restarts, it can technically have item be anything so prob just leave it documented as Object.

Reply:
Does this apply for the whole of `item` or just `item.value`?

Reply:
Should be applicable for the entirety of item.